explainer.md

```diff
@@ -0,0 +1,329 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**
+
+- [Background: assistive technology and the accessibility tree](#background-assistive-technology-and-the-accessibility-tree)
+  - [Accessibility node properties](#accessibility-node-properties)
+- [Existing accessibility APIs](#existing-accessibility-apis)
+  - [Native platform accessibility APIs](#native-platform-accessibility-apis)
+  - [Gaps in the web platform's accessibility story](#gaps-in-the-web-platforms-accessibility-story)
+  - [Audience for the proposed API](#audience-for-the-proposed-api)
+- [The Accessibility Object Model](#the-accessibility-object-model)
+  - [Exploring the accessibility tree](#exploring-the-accessibility-tree)
+- [Use cases](#use-cases)
+  - [Use DOM node references instead of IDREFs](#use-dom-node-references-instead-of-idrefs)
+  - [Create virtual accessibility trees](#create-virtual-accessibility-trees)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
```

Comment:
I wonder if it would be useful to have a couple sentences at the beginning explaining what this thing is. Similar to what's already in the README but for those folks who might get linked directly to this document